app/views/site/covid19.html.erb

```diff
@@ -36,7 +36,7 @@
         <a href="#" id="covid-contact-link">contact us</a>.
        </span>
       <div class="tab-content top-pad">
-        <div id="home-page-content"></div>
+        <div id="home-view"></div>
```

Comment:
Is this supposed to have been changed? I don't see a corresponding hook for this id anywhere in the PR

Reply:
Good catch -- I had changed this in both indexes, decided against it, and rolled back one index but not the other.